opba-facade-protocol-api-shared/src/main/java/de/adorsys/opba/protocol/api/common/CurrentBankProfile.java

```diff
@@ -36,4 +36,9 @@
      * Preferred SCA approach for this ASPSP.
      */
     Approach getPreferredApproach();
+
+    /**
+     * ASPSP forces preferred SCA approach.
+     */
+    Boolean getForcePreferredApproach();
```

Comment:
Why do we have it as a nullable type - we can only force approach by sending header or not. Suggest to use as the non-null type

Reply:
done